### ISSUE TITLE
Ignoring asset only npm packages

### ISSUE THREAD

**Issue by rgbkrk** [flowtype/flow-typed#356]
We have a package that only has fonts (to help trim our Electron package). It has no entrypoint (recently taken out of our package.json). How can we start using `flow-typed` yet ignore this one package?

```
$ flow-typed install
• Found flow-bin@v0.33.0 installed. Installing libdefs compatible with this version of Flow...
• Found 72 dependencies in package.json. Searching for libdefs...
• Installing 7 libdefs...
  • flow-typed/npm/moment_v2.x.x.js already exists!
    └> Use --overwrite to overwrite the existing libdef.
  • flow-typed/npm/react-dnd-html5-backend_v2.1.x.js already exists!
    └> Use --overwrite to overwrite the existing libdef.
  • flow-typed/npm/react-dnd_v2.x.x.js already exists!
    └> Use --overwrite to overwrite the existing libdef.
  • flow-typed/npm/react-redux_v4.x.x.js already exists!
    └> Use --overwrite to overwrite the existing libdef.
  • flow-typed/npm/redux_v3.x.x.js already exists!
    └> Use --overwrite to overwrite the existing libdef.
  • flow-typed/npm/react-addons-test-utils_v15.x.x.js already exists!
    └> Use --overwrite to overwrite the existing libdef.
  • flow-typed/npm/enzyme_v2.x.x.js already exists!
    └> Use --overwrite to overwrite the existing libdef.
UNCAUGHT ERROR: Error: Cannot find module 'nteract-assets' from '/Users/kylek/code/src/github.com/nteract/nteract'
    at /Users/kylek/.nvm/versions/node/v6.7.0/lib/node_modules/flow-typed/node_modules/resolve/lib/async.js:46:17
    at process (/Users/kylek/.nvm/versions/node/v6.7.0/lib/node_modules/flow-typed/node_modules/resolve/lib/async.js:173:43)
    at ondir (/Users/kylek/.nvm/versions/node/v6.7.0/lib/node_modules/flow-typed/node_modules/resolve/lib/async.js:188:17)
    at load (/Users/kylek/.nvm/versions/node/v6.7.0/lib/node_modules/flow-typed/node_modules/resolve/lib/async.js:69:43)
    at onex (/Users/kylek/.nvm/versions/node/v6.7.0/lib/node_modules/flow-typed/node_modules/resolve/lib/async.js:92:31)
    at /Users/kylek/.nvm/versions/node/v6.7.0/lib/node_modules/flow-typed/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:123:15)
```

**Comment by rgbkrk** [comment]
To get around this temporarily:

```
touch node_modules/nteract-assets/index.js
```

Even though we don't need it in the packages. What should be added to the package or adapted here to handle this? I got rid of the `main`, thinking that's what would lead `flow` somewhere.

**Comment by nikolay** [comment]
@rgbkrk This is a showstopper for many. We could have many modules with this issue and doesn't make sense to touch a bunch of files all the time when we install modules. Any module that doesn't meet the criteria should be skipped silently or there could be an optional warning.

**Comment by jeffmo** [comment]
I've just published `flow-typed@2.0.0-beta.11` with https://github.com/flowtype/flow-typed/commit/d295e7d150309b8e950ee623ef7fb05df18cc4e3, which should fix this.
Please try it out and let me know if you have any more issues!

(NOTE: This is the same issue as https://github.com/flowtype/flow-typed/issues/338)

**Comment by rgbkrk** [comment]
In the meantime I added a dummy `index.js` to make this work cleanly.

I'll certainly try out the beta, thanks for hopping on this!
